### DISCUSSION TITLE
Visual Studio 2019 - Can't run project after uninstalling .NET6

### DISCUSSION THREAD

**Discussion by BarPupko** [AvaloniaUI/Avalonia#8714] [Q&A] [answered]
Hi Guys,
I had on my Machine Version 6 of .NET
after uninstalling version 6 , I tried to create a new Avalonia MVVM Project, when I try to run the project the get the next error.
https://imgur.com/XsE8qk8
Thank you for your support, Really Appreciate it.

**Comment by MihaMarkic** [comment]
While I can't say for sure, why don't you simply install .net 6 again?

**Reply by BarPupko** [reply]
Hi, Thank you for responding.
I reinstalled the.NET 6 version, but when I try to load the project, I still receive the same issue.
I'm looking for a way to connect the project to the run program.
https://imgur.com/0PRcWai

**Reply by MihaMarkic** [reply]
Can you upload the project somewhere to take a look at?

**Reply by BarPupko** [reply]
Yeah sure I added to my repos
https://github.com/BarPupko/AvaloniaApplication2

Thank you

**Comment by maxkatz6** [comment, marked as answer]
Couple of problems:
1. Microsoft doesn't support .NET 6 on VS 2019. You better to upgrade Visual Studio in your team if you want to follow it.
2. .NET 6 SDK needs to be installed if you want to use target framework "net6.0"
3. Avalonia doesn't really care about your specific .NET version, it only needs to be netstandard2 compatible. Which means, you can use older .NET version - 3.1 or 5.

You can change target version to lower in project settings or csproj file.

**Reply by BarPupko** [reply]
Thank you for your assistance. I installed Visual Studio 2022, as well as a newer version of.NET 7 (Preview), and it worked!

best wishes :)
